src/routes/events/handlers.js

```diff
@@ -79,7 +98,7 @@ export const getHandler = async (req, res) => {
 
     const auth = await getEventAuthorization(req, res, event);
 
-    if (!auth.canRead()) {
+    if (!auth.canRead() && !auth.isCollaborator()) {
```

Comment:
Should the isCollaborator be specific to the TR reports or will this also indicate if they are a collaborator on the AR?

Reply:
this auth refers only to the TR, its a ref to the EventPolicy